terraform-config.tf

```diff
@@ -0,0 +1,39 @@
+variable "ami_id" {
+  type = string
+}
+
+variable "region" {
+  description = "The AWS region to use for resources"
+  type        = string
+  default     = "us-east-1"
+}
+
+provider "aws" {
+  region = var.region
+}
+
+data "aws_security_group" "selected" {
+  id = "sg-037fd498b332442c1"
+}
+
+resource "aws_instance" "test" {
```

Comment:
These are predefined types, otherwise, you could change `test` by `github_action_test` or `CI_test` or something maybe more precise WDYT?